system_metrics_collector/src/system_metrics_collector/main.cpp

```diff
@@ -25,6 +25,8 @@
 #include "../../src/system_metrics_collector/linux_cpu_measurement_node.hpp"
 #include "../../src/system_metrics_collector/linux_memory_measurement_node.hpp"
 
+static constexpr const char STATISTICS_TOPIC_NAME[] = "system_metrics";
```

Comment:
Nit: You used anonymous namespaces elsewhere, for consistency this should also be in an anonymous namespace.

Reply:
Done.